clang/test/Sema/constant-builtins-all-args-evaluated.cpp

```diff
@@ -0,0 +1,39 @@
+// RUN: %clang_cc1 -fsyntax-only -verify %s
+// expected-no-diagnostics
+
+constexpr int increment(int& x) {
+  x++;
+  return x;
+}
+
+constexpr int test_clzg_0() {
+  int x = 0;
+  (void)__builtin_clzg(0U, increment(x));
```

Comment:
Random thought that popped up while in bed at 2 AM: I probably could have just used `++x` as second argument.